mysql-test/suite/innodb/t/ts_encrypt.test

```diff
@@ -0,0 +1,174 @@
+# InnoDB general tablespace encryption
+
+call mtr.add_suppression("Partition.*does not exist in the InnoDB internal data dictionary though");
+
+--source include/have_innodb.inc
+--source include/not_embedded.inc
```

Comment:
Does embedded support keyring plugins? I think I copied this one from table_encrypt_1 as it seemed logical to me